File: FragmentSample/app/src/main/java/com/example/fragmentsample/MainActivity.java
package com.example.fragmentsample;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ArticleReaderFragment articleReaderFragment =  new ArticleReaderFragment();

        Bundle bundle = new Bundle();
        articleReaderFragment.setArguments(bundle);

        getSupportFragmentManager().beginTransaction().replace(R.id.fragmentContainer, articleReaderFragment).commit();
    }
}

File: mlkit-android-image_labeling/barcode_scanning/CameraXApp/app/src/main/java/com/example/cameraxapp/MainActivity.kt
package com.example.cameraxapp

import androidx.appcompat.app.AppCompatActivity

import android.Manifest
import android.os.Bundle

class MainActivity : AppCompatActivity() {

    private val REQUEST_CODE_PERMISSIONS = 10

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }
}
